### DISCUSSION TITLE
Request: add .jpg to Image Save node options, filename extension stripping

### DISCUSSION THREAD

**Discussion by albamuth** [WASasquatch/was-node-suite-comfyui#192] [Ideas]
I am replacing a lot of .jpg's and not .jpeg's (trivial, I know) so it would be nice to have an option to save as .jpg as well.
Also, right now I have to do a text replace to strip out the original file's extension, would be nice as a built-in option but no big deal.

**Comment by WASasquatch** [comment]
Do you mean replace any `.ext` at the end of filename prefix?

Yeah I can add short alias.

**Reply by albamuth** [reply]
I'm batch loading "filename.jpg" with your Load Image node, sending the filename to the WAS Save Image node, but without intervention (replacing the ".jpg" with ""), it will end up as "filename.jpg.jpeg". A toggle to keep the extension or not would be nice.

**Reply by WASasquatch** [reply]
jpg and extension stripping is added: https://github.com/WASasquatch/was-node-suite-comfyui/commit/ad6b05a54487b0c5205f5329a49198b04ac98b9e
